master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -927,13 +932,19 @@ private[celeborn] class Master(
     sb.toString()
   }
 
-  override def listPartitionLocationInfo: String = throw new UnsupportedOperationException()
-
-  override def getUnavailablePeers: String = throw new UnsupportedOperationException()
-
-  override def isShutdown: String = throw new UnsupportedOperationException()
-
-  override def isRegistered: String = throw new UnsupportedOperationException()
+  override def exclude(id: String): String = {
+    val sb = new StringBuilder
+    sb.append("============================ Exclude Worker =============================\n")
+    val excludedWorker: WorkerInfo = WorkerInfo.fromUniqueId(id)
+    if (workersSnapShot.contains(excludedWorker)) {
+      statusSystem.updateWorkerExcludeMeta(excludedWorker)
```

Comment:
Need add method IMetadataHandler.handleWorkerExclude and implement in SingleMasterMetaManager/HAMasterMetaManager, you can refer exist implementation in these Manager.